scripts/jest/src/environment.js

```diff
@@ -0,0 +1,3 @@
+const isCI = Boolean(process.env.TF_BUILD);
```

Comment:
FYI It's true only for Azure Pipelines, if we will switch to GH Actions the variable will be different.

https://github.com/watson/ci-info/blob/20fae89d2bdeb0e5dd70e6a9e8d2647764e6ff04/vendors.json#L115-L122

Reply:
yes I know, reflecting what we have ATM, future plan is to use isCI package or have the logic within monorepo utils